### PR TITLE
Fix/clean dep audit fix

### PR DESCRIPTION
**Pull request by rfreitas**
# Problem

We have dev dependencies with security vulnerabilities.
It's hard to measure risk for dev dependencies, since it's unclear how much room they have for code injection, but anything that is pulled by webpack at production built time should be vulnerability free.

# Solution

1. Remove not used dependencies
2. Removed unused loaders
3. Upgraded to latest Less

## Left out

1. Surge.sh, it's not being used by webpack, it's only used for publishing the demo app which is public

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [x] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [n/a] Have new automated tests been implemented?
- [n/a] Have new manual tests been written down?
- [n/a] Have tests passed locally?
- [n/a] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-526-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-526-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-526-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-526-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://526-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by bartzet** [comment]
@rfreitas there are some minor conflicts to solve in this PR.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-526-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://526-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-526-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-526-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-526-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://526-pr-onfido-sdk-ui-onfido.surge.sh

File: webpack.config.babel.js [hunk 1]
```diff
@@ -226,10 +222,6 @@ const configDist = {
       {
         test: /\.(svg|woff2?|ttf|eot|jpe?g|png|gif)(\?.*)?$/i,
         use: ['file-loader?name=images/[name]_[hash:base64:5].[ext]']
-      },
-      {
-        test: /\.html$/,
-        use: ['html-loader?interpolate']
```
Comment (stefaniacardenas) [inline webpack.config.babel.js]: Why did you remove this and raw-loader?

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-526-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://526-pr-onfido-sdk-ui-onfido.surge.sh
